deployment/scripts/cluster.backup.sh

```diff
@@ -19,15 +19,8 @@ BACKUP_FOLDER=${BACKUP_FOLDER:-${SCRIPT_DIR}/../configurations/${CONFIGURATION}/
 mkdir -p ${BACKUP_FOLDER}
 
 # maintenance mode on
-${SCRIPT_DIR}/cluster.maintenance.sh on
-
-# shutdown database
-kubectl --kubeconfig=${KUBECONFIG} get deployment ocelot-neo4j -o json \
-    | jq '.spec.template.spec.containers[] += {"command": ["tail", "-f", "/dev/null"]}' \
-    | kubectl --kubeconfig=${KUBECONFIG} apply -f -
-
-# wait for the container to restart
-sleep 60
+# set Neo4j in offline mode (maintenance)
+${SCRIPT_DIR}/cluster.neo4j.sh offline-mode
```

Comment:
I changed `cluster.neo4j.sh` commands to: @ulfgebhardt 

- `scripts/cluster.neo4j.sh maintenance on`
- `scripts/cluster.neo4j.sh maintenance off`

If we would use it without `maintenance` it gets confusing with `cluster.maintenance.sh on/off`, because `scripts/cluster.neo4j.sh maintenance on` would become `scripts/cluster.neo4j.sh off` and the other command the other way around. So then the on/off for cluster maintenance would used together with the switched commdans off/on for the database.